css/index.less

```diff
@@ -0,0 +1,160 @@
+/* colors used throughout the page */
+@dark-brown: hsl(31, 49%, 10%);
+@light-brown: lighten(@dark-brown, 50%);
+@near-black: hsl(220, 6%, 10%);
+@red: hsl(0, 100%, 50%);
+@lighter-brown: lighten(@light-brown, 15%);
+/* set box-sizing, margin, padding, and color for html and body */
+html, body {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
```

Comment:
Box-sizing is pretty well adopted. Check caniuse.com to see if you need prefixes